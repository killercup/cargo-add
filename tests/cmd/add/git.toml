bin.name = "cargo-add"
args = ["add", "git-package", "--git", "http://localhost/git-package.git"]
status = "success"
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
