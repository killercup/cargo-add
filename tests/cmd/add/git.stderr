      Adding git-package (unknown version) to dependencies. 
